src/Services/NodeUptimeService.ts

```diff
@@ -14,17 +12,26 @@ export class NodeUptimeService {
     }
 
     public async fetchNodeUptime(nodeId: number, filter: string) {
-        return await NodeUptime.findAll({
-            raw: true,
-            where: {
-                nodeId,
-                updatedAt: {
-                    [Op.gte]:
-                        moment.utc().subtract(1, filter as unitOfTime.Base).format("YYYY-MM-DD HH:MM:ssZZ")
-                }
-            },
-            order: [['updatedAt', 'DESC']]
-        });
+        return await database.runQuery<NodeUptime>(
+            `select "isWorking",date_trunc('day', "updatedAt") "timePeriod", (
+                case when "isWorking" = false
+                then 'Node is down'
```

Comment:
Since this is an API, it's better and expected to use boolean (true/false) so frontend can parse it easier

Reply:
The idea was that the response also has a message that can be displayed, like ```    {
        "isWorking": false,
        "timePeriod": "2020-03-17T00:00:00.000Z",
        "nodeStatus": "Node is down"
    }```
Should I just leave the isWorking with timePeriod in the response?

Reply:
Yeah

Reply:
Also, please don't break current implementation by renaming variable, response should have the same structure